aas-gui/Frontend/aas-web-gui/src/components/SubmodelPlugins/HandoverDocumentation.vue

```diff
@@ -226,13 +226,9 @@ export default defineComponent({
             // prepare each DocumentVersion
             document.documentVersions.forEach((documentVersion: any) => {
                 // extract the DigitalFile
-                documentVersion.digitalFile = documentVersion.value.find((element: any) => {
-                    return element.idShort === "DigitalFile";
-                });
+                documentVersion.digitalFile = this.getElementBySemanticId("0173-1#02-ABI504", documentVersion);
                 // extract the PreviewFile
-                documentVersion.previewFile = documentVersion.value.find((element: any) => {
-                    return element.idShort === "PreviewFile";
-                });
+                documentVersion.previewFile = this.getElementBySemanticId("0173-1#02-ABI505", documentVersion);
```

Comment:
Shouldn't it be 0173-1#02-ABI505#001 instead of 0173-1#02-ABI505 (without version identifier #001)?